specifications/16-case-insensitive-in-operators.json

```diff
@@ -0,0 +1,200 @@
+{
+  "name": "16-case-insensitive-in-operators",
+  "state": {
+    "version": 2,
+    "features": [
+      {
+        "name": "F9.globalSegmentAndConstraint_IN",
+        "description": "With global segment and constraint both on",
+        "enabled": true,
+        "strategies": [
+          {
+            "name": "default",
+            "parameters": {},
+            "constraints": [
+              {
+                "contextName": "appName",
+                "operator": "IN",
+                "caseInsensitive": true,
+                "inverted": false,
+                "values": [
+                  "app"
+                ]
+              }
+            ],
+            "segments": [1]
```

Comment:
I don't think the segments add much value to these tests. They're just constraints in a dictionary and they add a lot of cognitive complexity to understanding this suite. They're already quite well covered in other suites so I would suggest either removing them or only including a single test with a segment

Reply:
Agreed